packages/desktop-gui/cypress/integration/release_notes_spec.js

```diff
@@ -0,0 +1,135 @@
+import { deferred } from '../support/util'
+
+describe('Release Notes', () => {
+  let ipc
+  let releaseNotes
+  let getReleaseNotes
+
+  beforeEach(() => {
+    let user
+
+    cy.fixture('user').then((theUser) => user = theUser)
+    cy.fixture('release_notes').then((theReleaseNotes) => releaseNotes = theReleaseNotes)
+
+    cy.visitIndex().then((win) => {
+      ipc = win.App.ipc
+
+      cy.stub(ipc, 'getCurrentUser').resolves(user)
+      cy.stub(ipc, 'externalOpen')
+      cy.stub(ipc, 'getOptions').resolves({ version: '1.0.0' })
+      cy.stub(ipc, 'updaterCheck').resolves('1.2.3')
+
+      getReleaseNotes = deferred()
+      cy.stub(ipc, 'getReleaseNotes').returns(getReleaseNotes.promise)
+
+      win.App.start()
+    })
+  })
+
+  it('opens modal after clicking "Learn more" on update notice', () => {
+    cy.get('.update-notice').contains('Learn more').click()
+    cy.get('.update-modal').should('be.visible')
+  })
+
+  it('shows loading spinner when loading release notes', () => {
+    cy.get('.update-notice').contains('Learn more').click()
+    cy.get('.update-modal .loader')
+  })
+
+  it('shows update instructions if no release notes for version', () => {
+    getReleaseNotes.resolve(null)
+    cy.get('.update-notice').contains('Learn more').click()
+    cy.contains('Update to Version 1.2.3').should('be.visible')
+  })
+
+  it('shows update instructions if getting release notes errors', () => {
+    getReleaseNotes.reject(new Error('something went wrong'))
+    cy.get('.update-notice').contains('Learn more').click()
+    cy.contains('Update to Version 1.2.3').should('be.visible')
+  })
+
+  describe('when there are release notes (with all fields included)', () => {
+    beforeEach(() => {
+      getReleaseNotes.resolve(releaseNotes)
+      cy.get('.update-notice').contains('Learn more').click()
+    })
+
+    it('shows release notes title', () => {
+      cy.contains(releaseNotes.title)
+    })
+
+    it('shows banner image', () => {
+      cy.get('.release-notes img').should('have.attr', 'src', 'https://placekitten.com/1000/200')
+      cy.get('.release-notes img').should('have.attr', 'width', '548')
+    })
+
+    it('shows content', () => {
+      cy.get('.release-notes .contents')
+      .shadow()
+      .find('h1')
+      .should('have.text', 'This is a great release')
```

Comment:
Doesn't look like a percy snapshot will work very well. Here's the result of trying one out: 

![Screen Shot 2020-10-30 at 10 49 25 AM](https://user-images.githubusercontent.com/1157043/97719649-b4c61480-1a9d-11eb-9319-ab7154ed3021.png)

It doesn't like emoji, the re-routed image, or shadow dom. I think I'll remove it for now and make a note to revisit in the future.

Unfortunately, even if we add shadow dom support to Cypress snapshots, it won't fix percy because they do their own snapshotting. It's just coincidence that we both use similar methods currently that don't support capturing shadow dom.